### ISSUE TITLE
Installer edits either not received or not reported correctly

### ISSUE THREAD

**Issue by Salvatos** [lutris/website#408]
So I just submitted an edit to an installer (PS Now) and the site said it was received and would be moderated. However when I look at my profile, under *Submitted contributions*, it says "You have no submitted contributions awaiting approval."

(And by the way, I’m a little worried that #398 hasn’t been reopened since it was never actually fixed.)

**Comment by tfks** [comment]
It does register, I did a new submit and when I check show unpublished installers it shows up. Another edit I did for an existing installer did get moderated. The change became visible in the installer script.

So it just doesn't show up in the user's profile.

**Comment by ZeroAbility** [comment]
> It does register, I did a new submit and when I check show unpublished installers it shows up. Another edit I did for an existing installer did get moderated. The change became visible in the installer script.
> 
> So it just doesn't show up in the user's profile.

I have noticed this with forking, as well. I submitted a fork for the DVD install for the following:

https://lutris.net/games/neverwinter-nights/

It shows two entries under unpublished but they appear to be the same as the Alternative and the older does not have my changes. I attempted another to see if I could get some kind of history of what I submitted and there isn't a feature that does that. It might be helpful to have a feature like that, otherwise git might make more sense. I am admittedly new to lutris so if there was a way to submit scripts through git, I apologize for my ignorance.
